### DISCUSSION TITLE
Remote background on linux agent

### DISCUSSION THREAD

**Discussion by my1770** [amidaware/tacticalrmm#1419] [General]
**Server Info (please complete the following information):**
 - OS: Ubuntu 20.04
 - Browser: chrome
 - RMM Version: v0.15.7

**Installation Method:**
  - [x] Standard
  - [ ] Docker

**Agent Info (please complete the following information):**
- Agent version:  Agent v2.4.4
- Agent OS: Ubuntu 22.04 x86_64 5.15.0-58-generic

![image](https://user-images.githubusercontent.com/120328340/215351358-96f3eb4c-4e0f-45a8-b273-6e375c6e85e8.png)

![image](https://user-images.githubusercontent.com/120328340/215351496-198f48bf-6ed0-4414-a4fd-9bad3ec146dd.png)

![image](https://user-images.githubusercontent.com/120328340/215351564-b82c18f2-a7d0-4278-9fcc-a2c24b150c64.png)

I'm trying to connect to the client via take control or remote background. An empty window opens.
Only PROCESSES is displayed.
If I install an agent from mesh, everything works in it.

**Comment by dinger1986** [comment]
How are you installing the linux agent? Do you have code signing?

**Comment by silversword411** [comment]
Terminal/File browser is working here (not remote desktop because this is a headless server), that's mesh integration. Look at your mesh. and see if that agent shows as online. 

![2023-01-29_150043 - brave_ntop1_-_0_PCR_-_Linux_Servers__Remote_Background_-](https://user-images.githubusercontent.com/13395348/215352706-2ead52b3-7b11-472f-84d3-cf043b7bd770.png)

Are you code signed? 

Are you doing #unsupported proxies or something with your config?

**Reply by my1770** [reply]
I understand the difference between terminal and take control. In both cases, it does not connect. I connect to the client via ssh without any problems.
I set the agent manually. All services work, except for the terminal, the file manager.

**Reply by bbrendon** [reply]
Log into mesh directly and see. Maybe you don't have access to the agent.

**Reply by my1770** [reply]
They did not appear in mesh, an agent from mesh installed. Works from there

**Reply by dinger1986** [reply]
How are you installing the agent? Do you have code signing?
